dev/PushNotifications/PushNotifications.idl

```diff
@@ -39,11 +39,9 @@ namespace Microsoft.Windows.PushNotifications
         // Initialize using a Registration option and optionally defined parameters like manifest defined activatorId
         // 1) If kind = PushTrigger is specified, only the Push Trigger will be Registered with Background Infra
         // 2) If kind = ComActivator is specified, the Background Task component will be Registered as an in-process COM server
+        // 3) If kind = ProtocolActivator is specified, the LRP will CoCreateInstance the application in background scenarios
```

Comment:
Need to revert this change but we should tell packaged devs to call this constructor with protocol activator & comClsid if push trigger is unsupported. Unpackaged apps will be using the other one I removed and we can assume GUID_NULL. What do you think here? @hulumane